config/pepperConfig.js.ctmpl

```diff
@@ -39,7 +39,7 @@ var DDP_ENV = {
     DDP_ENV['platformGAToken'] = "UA-108484823-1";
 {{end}}
 
-{{if (not (or (or (or (eq $study_guid "testboston") (eq $study_guid "PRION")) (eq $study_guid "atcp")) (eq $study_guid "RGP")))}}
+{{if (eq $study_guid "ANGIO" "cmi-brain" "cmi-esc" "cmi-mbc" "cmi-mpc" "CMI-OSTEO")}}
```

Comment:
[Docs](https://golang.org/pkg/text/template/#hdr-Functions) say `eq` will do equality testing of the first argument to all the other arguments, which is what we want.
> For simpler multi-way equality tests, eq (only) accepts two or more arguments and compares the second and subsequent to the first, returning in effect
> `arg1==arg2 || arg1==arg3 || arg1==arg4 ...`